distribution/apim-accelerator/resources/repository/conf/deployment.toml

```diff
@@ -418,6 +418,26 @@ policy.limit.time_unit = "min"
 patient_id_uri = "http://wso2.org/claims/patientId"
 patient_id_key = "patientId"
 
+[[healthcare.backend.auth]]
```

Comment:
Shall we comment these by default as well, since it should be configured to the actual values by the customer

Reply:
done https://github.com/wso2/healthcare-accelerator/commit/2baf7e5a6442a5828c556972e262416fefba19b3